### PR TITLE
8.2.1 changelog and version number

### PR DESCRIPTION
**Pull request by jospoortvliet**
@cmonteroluque I've summarized the changelog, 89 items for CE alone seemed a bit much ;-)

@jnweiger Once we have the new VM number, let me know. We currently have this:
```$VERSIONS_VM = '8.2.0-2.1-201510201921';```

### CODE REVIEW

File: page-changelog.php [hunk 1]
```diff
@@ -3,6 +3,80 @@
...
+</ul>
+</ul>
+Download: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2">owncloud-8.2.1.tar.bz2</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip">owncloud-8.2.1.zip</a></br>
+MD5: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.md5">owncloud-8.2.1.tar.bz2.md5</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.md5">owncloud-8.2.1.zip.md5</a></br>
+SHA256: <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.sha256">owncloud-8.2.1.tar.bz2.sha256</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.sha256">owncloud-8.2.1.zip.sha256</a></br>
+PGP (<a href="https://owncloud.org/owncloud.asc">Key</a>): <a href="https://download.owncloud.org/community/owncloud-8.2.1.tar.bz2.asc">owncloud-8.2.1.tar.bz2.asc</a> or <a href="https://download.owncloud.org/community/owncloud-8.2.1.zip.asc">owncloud-8.2.1.zip.asc</a></br>
+Packages: <a href="http://software.opensuse.org/download/package?project=isv:ownCloud:community:8.1&package=owncloud">on the Open Build Service</a></br>
```
Comment (RealRancor) [inline page-changelog.php]: Should be https://download.owncloud.org/download/repositories/stable/owncloud/ or https://download.owncloud.org/download/repositories/8.2/owncloud/

Comment (jnweiger) [inline page-changelog.php]: @jospoortvliet there is an old isv:ownCloud:community:8.1 link there. Can you please fix that too, as @RealRancor suggests?

Comment (jospoortvliet) [inline page-changelog.php]: Done.

File: page-changelog.php [hunk 2]
```diff
@@ -3,6 +3,80 @@
...
+<li>Do not register sidebar panels when no sidebar</li>
+<li>Update process: state which step we are going to start and warn if it might be slow</li>
+</ul>
+<li>User Management and LDAP fixes:</li>
+<ul>
+<li>LDAP fixes for quota, user mapper, initial user creation and connectivity</li>
+<li>Fix group admin settings, group assignment when group name is a number</li>
+<li>Enable proper CardDAV cookie authentication</li>
+</ul>
+<li>Documentation, reliability and stability fixes</li>
```
Comment (phil-davis) [inline page-changelog.php]: All the other sub-heading text ends in a colon, except this one here. The text parser in my brain had a stack dump when it got to here :)

File: page-changelog.php [hunk 3]
```diff
@@ -3,6 +3,80 @@
 </div>
 <p>We recommend you follow our <a href="/release-channels">Release Channels</a> to always have the latest ownCloud Server version which fits your needs.</p>
 
+<h3>Version 8.2.1 <small>November 17 2015</small></h3>
+<ul>
+<li>Sharing improvements:</li>
+<ul>
```
Comment (phil-davis) [inline page-changelog.php]: Looks like this should be a /ul tag to end the ul - and then all the other ul tag pairs seem to be out-of-sync.

File: page-changelog.php (hunk 2 above)
Comment (jospoortvliet) [inline page-changelog.php]: lol

fixed

File: page-changelog.php (hunk 3 above)
Comment (jospoortvliet) [inline page-changelog.php]: no-no, this is correct. At least it works ;-)

**Pull request merged**